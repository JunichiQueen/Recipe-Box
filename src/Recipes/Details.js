import React, { Component } from 'react';
import axios from 'axios';

class Details extends Component {

    // deleteRecipt = () => {
    //     axios.
    // }

    render(){
        return(
            <div>
                <p>{this.props.name}</p>
                <p>{this.props.description}</p>
                <p>{this.props.ingredients}</p>
            </div>
        )
    }
}

export default Details;