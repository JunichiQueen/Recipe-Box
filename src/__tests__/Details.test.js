import React from 'react';
import { shallow } from "enzyme";
import renderer from 'react-test-renderer'

import Details from '../Recipes/Details.js';

it('renders without crashing', () => {
    shallow(<Details />);
})

it('snapshot Details', () => {
    const wrapper = renderer.create(<Details />).toJSON();
    expect(wrapper).toMatchSnapshot();
})