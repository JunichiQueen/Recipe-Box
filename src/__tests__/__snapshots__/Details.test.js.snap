// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot Details 1`] = `
<div>
  <p />
  <p />
  <p />
  <img />
  <button
    aria-label={null}
    className="btn btn-outline-danger"
    onClick={[Function]}
    type="button"
  >
    Delete Recipe
  </button>
  <div>
    <form
      onSubmit={[Function]}
    >
      <p>
        RecipeName
      </p>
      <input />
      <p>
        Description
      </p>
      <input />
      <p>
        Ingredients
      </p>
      <input />
      <p>
        Image
      </p>
      <input />
      <button
        aria-label={null}
        className="btn btn-outline-warning"
        onClick={[Function]}
        type="submit"
      >
        Update Recipe
      </button>
    </form>
  </div>
</div>
`;
